src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -2282,6 +2284,18 @@ public String getFileExtension()
             return index == 0 || index == fileName.length() ? null : fileName.substring(index);
         }
 
+        /**
+         * The Content-Type of this file.
+         * <br>This is the  <a href="https://en.wikipedia.org/wiki/Media_type" target="_blank">Media type</a> of the file that would be used in an HTTP request or similar.
```

Comment:
```suggestion
         * <br>This is the <a href="https://en.wikipedia.org/wiki/Media_type" target="_blank">Media type</a> of the file that would be used in an HTTP request or similar.
```

Reply:
@MinnDevelopment This doesn't appear like it was actually incorporated. Why was it marked `resolved`?

Reply:
Because it changes nothing of value.